Firestore/core/test/unit/remote/bloom_filter_test.cc

```diff
@@ -132,6 +142,131 @@ TEST(BloomFilterTest,
   }
 }
 
+class BloomFilterGoldenTest : public ::testing::Test {
+ public:
+  void RunGoldenTest(std::string& test_file) {
+    size_t start_pos = test_file.find("bloom_filter_proto");
+    if (start_pos == std::string::npos) {
+      return;
+    }
+    std::string result_file = test_file;
+    result_file.replace(start_pos, sizeof("bloom_filter_proto") - 1,
+                        "membership_test_result");
+
+    json test_file_json = ReadFile(test_file);
+    json result_file_json = ReadFile(result_file);
+
+    nlohmann::json bits = reader.OptionalObject("bits", test_file_json, {});
+    std::string bitmap = reader.OptionalString("bitmap", bits, "");
+    int padding = reader.OptionalInt("padding", bits, 0);
+    int hash_count = reader.OptionalInt("hashCount", test_file_json, 0);
+    std::string decoded;
+    absl::Base64Unescape(bitmap, &decoded);
+    std::vector<uint8_t> decoded_map(decoded.begin(), decoded.end());
+    BloomFilter bloom_filter = BloomFilter(decoded_map, padding, hash_count);
+
+    std::string membership_result =
+        reader.OptionalString("membershipTestResults", result_file_json, "");
+
+    for (size_t i = 0; i < membership_result.length(); i++) {
+      bool expectedResult = membership_result[i] == '1';
+      bool mightContainResult = bloom_filter.MightContain(
+          GOLDEN_DOCUMENT_PREFIX_ + std::to_string(i));
+
+      EXPECT_EQ(mightContainResult, expectedResult);
+    }
+  }
+
+ private:
+  std::string GOLDEN_DOCUMENT_PREFIX_ =
```

Comment:
The `GOLDEN_DOCUMENT_PREFIX_` and `GOLDEN_TEST_FOLDER_` member variables should be declared `const` and `static` since they are the same for every instance of the class. Also, the naming convention would have their names be `kGoldenDocumentPrefix` and `kGoldenTestFolder`, respectively.

Reply:
Does't allow me to do "static const". is it because I am not doing this in header file?
<img width="1505" alt="Screenshot 2023-02-28 at 3 35 21 PM" src="https://user-images.githubusercontent.com/107142260/222007439-7dd64133-d9fa-40ed-8501-f779d421a733.png">

Reply:
if the member type is build in types like int, char, its okay to do "static" tho. 🤔

Reply:
Discussed offline.